library/src/main/java/com/novoda/downloadmanager/FilePathCreator.java

```diff
@@ -1,9 +1,13 @@
 package com.novoda.downloadmanager;
 
-public class FilePathCreator {
+public final class FilePathCreator {
 
     private static final String UNKNOWN = "unknown";
 
+    private FilePathCreator() {
```

Comment:
should the constructor be after all class fields?

Reply:
feb0b80